.github/workflows/checks-and-tests.yml

```diff
@@ -218,6 +219,7 @@ jobs:
     with:
       integration_name: 'tensorflow'
       deprecated: false
+      python_matrix: "['3.8', '3.9', '3.10', '3.11']"
```

Comment:
I'm wondering if we could remove the unit tests and CI settings for testing `tensorflow` integration and `mxnet` integration since these features are deprecated.

Reply:
Thank you for your comment. I have added `deprecated_class` decorator to MXNet integration.